app/services/jbpm_process_service.rb

```diff
@@ -39,6 +39,12 @@ def process_options
     options
   end
 
+  def validate(group)
+    raise Exceptions::RBACError, "Group #{group.uuid} doesn't have needed roles" unless group.with_approval_role?
```

Comment:
Can we print group name (or both) instead? The text can be straight `Group does not have approval role".